crates/workspace/src/persistence.rs

```diff
@@ -767,6 +767,12 @@ impl WorkspaceDb {
         let dev_server_projects = self.dev_server_projects()?;
 
         for (id, location, order, dev_server_project_id) in self.recent_workspaces()? {
+            let temp_order = order.clone();
```

Comment:
1. `.clone()` is not needed here
2. This seems to be the wrong place to fix things in, but maybe I misunderstand the root cause.
If this is the right fix, I think it belongs to https://github.com/zed-industries/zed/blob/1e39d407c24dbe5a0eeee3f12b2138129bcb95fa/crates/workspace/src/persistence/model.rs#L99

I understand it's not possible to fix it directly there though (ergo hoped for @eth0net to chime in with better ideas), at least it's obvious that there's more than one `SELECT .. local_paths_order, ..` around — seems that we have to fix them all too?